scripts/scalalib-patch-tool.sc

```diff
@@ -1,6 +1,11 @@
-import $ivy.`com.lihaoyi::ammonite-ops:2.3.8`, ammonite.ops._, mainargs._
```

Comment:
Maybe this could be in another PR so you could get it in faster. It is not related correct?

Reply:
Looks like you need to run `scripts/scalafmt` for CI happiness.

Reply:
> Maybe this could be in another PR so you could get it in faster. It is not related correct?

Yeah :+1:
It was required to update patches during development for some reason https://github.com/scala-native/scala-native/issues/3220#issuecomment-1521588637 🤔 
But now it turned out it's no longer needed, I'll remove this change and open a separate PR.